mbway/src/main/java/com/adyen/checkout/mbway/MbWayView.kt

```diff
@@ -44,8 +44,6 @@ internal class MbWayView @JvmOverloads constructor(
     init {
         orientation = VERTICAL
 
-        LayoutInflater.from(context).inflate(R.layout.mbway_view, this, true)
```

Comment:
Oh wow I'm amazed that didn't cause any issues

Reply:
It did 😂 The view was duplicated